src/sentry/static/sentry/app/components/sidebar/legacyOnboardingStatus.tsx

```diff
@@ -16,14 +16,6 @@ type Props = CommonSidebarProps & {
   org: Organization;
 };
 
-function recordAnalytics(currentPanel: string, orgId: string) {
-  const data =
-    currentPanel === 'todos'
-      ? {eventKey: 'onboarding.wizard_opened', eventName: 'Onboarding Wizard Opened'}
-      : {eventKey: 'onboarding.wizard_closed', eventName: 'Onboarding Wizard Closed'};
```

Comment:
I removed the `wizard closed` event since it's actually not that useful. We don't track the session, so we can't actually see how long the wizard is open since you need to associate a close with an open, which we don't.